app/sim_bench_test.go

```diff
@@ -27,7 +27,7 @@ func BenchmarkFullAppSimulation(b *testing.B) {
 		}
 	}()
 
-	app := NewWasmApp(logger, db, nil, true, simapp.FlagPeriodValue, map[int64]bool{}, interBlockCacheOpt())
+	app := NewWasmApp(logger, db, nil, true, simapp.FlagPeriodValue, wasm.DefaultEnabledProposals, map[int64]bool{}, interBlockCacheOpt())
```

Comment:
Maybe we rename `wasm.DefaultEnabledProposals` to `wasm.EnableAllProposals` and make a `wasm.DisableAllProposals` (empty slice). It will make reading the high-level app code clearer (is default all, none or some?)

Happy to use `EnableAllProposals` in all tests (like default now)